containers/SingleStation.js

```diff
@@ -7,53 +7,53 @@ export default class SingleStation extends React.Component {
     this.state = {
       trains: [],
       loading: true,
-      refreshing: false
+      refreshing: false,
     };
   }
 
   componentDidMount() {
     this.fetchTrains()
   }
 
-  fetchTrains = () => {
-    let myRequest = "https://api.wmata.com/StationPrediction.svc/json/GetPrediction/All"
-    let myHeaders = new Headers();
-    myHeaders.append("api_key", "a0363af8b5ab489bb2ce9479697aa70a");
-    let myInit = {method: "Get", headers: myHeaders};
-    fetch(myRequest, myInit)
-      .then(res => res.json())
-      .then(json => this.setState({ trains: json.Trains, loading: false }))
-      .catch((err) => console.log('err:', err))
-  }
-
   handleRefresh = () => {
-    this.setState({refreshing: true, loading: true});
+    this.setState({ refreshing: true, loading: true });
     this.fetchTrains()
-    this.setState({refreshing: false});
+    this.setState({ refreshing: false });
   }
 
   getStationTrainsByGroup = (trains, stationCode, group) => {
     const stationTrains = this.filterTrainsByStation(
       trains,
-      stationCode
+      stationCode,
     )
     return this.filterTrainsByGroup(stationTrains, group)
   }
 
   filterTrainsByStation = (trains, stationCode) => {
-    return trains.filter(train => train.LocationCode === stationCode)
+    trains.filter(train => train.LocationCode === stationCode)
   }
 
   filterTrainsByGroup = (trains, group) => {
-    return trains.filter(train => train.Group === group)
+    trains.filter(train => train.Group === group)
+  }
+
+  fetchTrains = () => {
+    const myRequest = 'https://api.wmata.com/StationPrediction.svc/json/GetPrediction/All'
+    const myHeaders = new Headers();
+    myHeaders.append('api_key', 'a0363af8b5ab489bb2ce9479697aa70a');
+    const myInit = { method: 'Get', headers: myHeaders };
+    fetch(myRequest, myInit)
+      .then(res => res.json())
+      .then(json => this.setState({ trains: json.Trains, loading: false }))
+      .catch(err => console.log('err:', err))
```

Comment:
Unexpected console statement.

[Read more about it here.](http://eslint.org/docs/rules/no-console)